plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/configrepo/v2/JsonMessageHandler2_0.java

```diff
@@ -71,18 +72,14 @@ private ParseDirectoryMessage prepareMessage_1(String destinationFolder, Collect
         return requestMessage;
     }
 
-    class ResponseScratch {
-        public Integer target_version;
-    }
-
     private ResponseScratch parseResponseForMigration(String responseBody) {
         return new GsonBuilder().create().fromJson(responseBody, ResponseScratch.class);
     }
 
     @Override
-    public String responseMessageForPipelineExport(String responseBody) {
+    public ExportedConfig responseMessageForPipelineExport(String responseBody, Map<String, String> headers) {
```

Comment:
Mentioned this elsewhere, but this embodies the reason why we added `headers` to the interface signature; need response headers to bind metadata and content from `pipeline-export` together into `ExportedConfig` objects.